.github/workflows/deploy-dev.yml

```diff
@@ -12,20 +12,13 @@ on:
 jobs:
   call-workflow:
     if: "(github.ref_name == 'master') || contains(github.event.head_commit.message, 'deploy:dev') || contains(github.ref, 'feature/')"
-    uses: navikt/pam-deploy/.github/workflows/deploy-dev.yml@v7
+    uses: navikt/pam-deploy/.github/workflows/deploy-next-js-dev.yml@next-js-deploy
```

Comment:
Er @next-js-deploy riktig versjon nummer?

Reply:
Nej, ska ändra det. Har inte mergeat in den där än :)